com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/schema.py

```diff
@@ -21,6 +21,24 @@
 import tokenize
 
 
+def is_common(schema):
+    """
+    Is `schema` an common schema
+    Args:
+        schema: Scheme to test.
+
+    Returns:
+        bool: ``True`` if schema is a common schema, otherwise ``False``.
+
+    """
+    if isinstance(schema, StreamSchema):
+        return schema.schema() in _SCHEMA_COMMON
+    if isinstance(schema, CommonSchema):
+        return True
+    if isinstance(schema, str):
+        return is_common(StreamSchema(schema))
+    raise TypeError()
```

Comment:
Rather than raise a TypeError, shouldn't this just return `False`?

Reply:
I'll change that. Thanks,